Sdk/Speckle.Connectors.Utils/Instances/InstanceObjectsManager.cs

```diff
@@ -1,9 +1,29 @@
-﻿using Speckle.InterfaceGenerator;
+﻿using System.Diagnostics.CodeAnalysis;
 using Speckle.Sdk.Models.Instances;
 
 namespace Speckle.Connectors.Utils.Instances;
 
-[GenerateAutoInterface]
+public interface IInstanceObjectsManager<THostObjectType, TAppIdMapValueType>
```

Comment:
If we're not auto generating it anymore, can we move it to it's own file?

Reply:
will do